Marlin/ultralcd.cpp

```diff
@@ -3636,7 +3666,17 @@ void kill_screen(const char* lcd_msg) {
 
     void lcd_advanced_pause_init_message() {
       START_SCREEN();
-      STATIC_ITEM(MSG_FILAMENT_CHANGE_HEADER, true, true);
+      switch (advanced_pause_mode) {
+        case ADVANCED_PAUSE_MODE_LOAD_FILAMENT:
+          STATIC_ITEM(MSG_FILAMENT_CHANGE_HEADER_LOAD, true, true);
+          break;
+        case ADVANCED_PAUSE_MODE_UNLOAD_FILAMENT:
+          STATIC_ITEM(MSG_FILAMENT_CHANGE_HEADER_UNLOAD, true, true);
+          break;
+        default:
+          STATIC_ITEM(MSG_FILAMENT_CHANGE_HEADER_PAUSE, true, true);
+          break;
+      }
```

Comment:
As this is repeated several times, it should be made it a macro…
```cpp
#define _ADVANCED_PAUSE_HEADER() \
  do { switch (advanced_pause_mode) {                             \
    case ADVANCED_PAUSE_MODE_LOAD_FILAMENT:                       \
      STATIC_ITEM(MSG_FILAMENT_CHANGE_HEADER_LOAD, true, true);   \
      break;                                                      \
    case ADVANCED_PAUSE_MODE_UNLOAD_FILAMENT:                     \
      STATIC_ITEM(MSG_FILAMENT_CHANGE_HEADER_UNLOAD, true, true); \
      break;                                                      \
    default:                                                      \
      STATIC_ITEM(MSG_FILAMENT_CHANGE_HEADER_PAUSE, true, true);  \
      break;                                                      \
  } } while(0)
```
…and then they can all be replaced by:
```cpp
_ADVANCED_PAUSE_HEADER();
```